src/colors.js

```diff
@@ -16,6 +16,7 @@
 
 export default {
   bg: '#1D1D1D',
+  bg_button: '#48acf0',
   bg_text: '#FFFFFF',
   bg_text_sec: '#B4B5B0',
   bg_positive: '#72E762',
```

Comment:
Can we remove this `bg_positive` if not used anywhere anymore?

Reply:
I guess you mean `bg_text_positive`, `bg_positive` is in use now

Reply:
wait a minute, let me check where are these `bg_positive` are used...